WORKSPACE

```diff
@@ -0,0 +1,40 @@
+# Declare the local Bazel workspace.
+# This is *not* included in the published distribution.
+workspace(
+    # If your ruleset is "official"
+    # (i.e. is in the bazelbuild GitHub org)
+    # then this should just be named "rules_mylang"
+    # see https://docs.bazel.build/versions/main/skylark/deploying.html#workspace
+    name = "com_myorg_rules_mylang",
+)
+
+load(":internal_deps.bzl", "rules_mylang_internal_deps")
+
+# Fetch deps needed only locally for development
+rules_mylang_internal_deps()
+
+load("//mylang:repositories.bzl", "mylang_register_toolchains", "rules_mylang_dependencies")
+
+# Fetch our "runtime" dependencies which users need as well
+rules_mylang_dependencies()
```

Comment:
```suggestion
# Fetch dependencies which users need as well
rules_mylang_dependencies()
```
Nit-picking: I know "runtime" was quoted, but still, to me it seems clearer and less confusing to just omit it.